pages/authentication/auth.css

```diff
@@ -0,0 +1,125 @@
+.main-section {
+  width: 100%;
+  min-height: 80vh;
+  height: 100%;
+  padding-top: 40px;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  background: radial-gradient(var(--secondary-color), var(--primary-color));
+}
+
+.section-form,
+.section-img {
+  flex: 1;
+}
+
+.section-form,
+.section-img {
+  padding: 40px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+form {
```

Comment:
Just a suggestion: Add a border-radius of a few pixels

Reply:
ok.. will add